internal/node/node_repositories.bzl

```diff
@@ -197,7 +197,7 @@ def _prepare_node(repository_ctx):
     """
 
     # TODO: Maybe we want to encode the OS as a specific attribute rather than do it based on naming?
-    is_windows = is_windows_os(repository_ctx) or "_windows_" in repository_ctx.attr.name
+    is_windows = "_windows_" in repository_ctx.attr.name
```

Comment:
Yeah so it was the issue we talked about, thanks @gregmagolan for the quick fix.

Reply:
No prob. The trouble is getting coverage on Windows. buildkite fails on Windows when adding rules_docker because of some python issue 🙄

Reply:
I don't think there is much documentation on it, but I think for windows you might have to explicitely pass in the `autodetecting_toolchain_nonstrict` - I could not find any documentation on it but just the comment here: https://github.com/bazelbuild/bazel/blob/00ec2e76e36ea540d8c26c5b9608cd38414b7d70/tools/python/pywrapper_template.txt#L87

Other than that rules_docker is just migrating everything away from python to go, but I think finishing that up is still a few weeks away.

Reply:
Actually finding there still is: https://github.com/bazelbuild/bazel/issues/7844

Reply:
Seems the python toolchain for windows is still in a poor state and we would need to define a python toolchain manually as is done here: https://github.com/irengrig/bazel/commit/10840c458fb1f114a4da927e68b4880840cd7703#diff-c467ef71bdf4e156e40ab02be09dccd5R368

Reply:
Ahhh. Thanks for looking that up. #909 is reproducible without docker so simplified this PR to avoid the docker rules for now.